src/hotspot/share/utilities/vmError.cpp

```diff
@@ -590,9 +590,6 @@ static void report_vm_version(outputStream* st, char* buf, int buflen) {
 #endif
                  UseCompressedOops ? ", compressed oops" : "",
                  UseCompressedClassPointers ? ", compressed class ptrs" : "",
-                 LockingMode == LM_MONITOR ? ", lm_monitors" :
-                 LockingMode == LM_LEGACY ? ", lm_legacy" :
-                 LockingMode == LM_LIGHTWEIGHT ? ", lm_lightweight" : "",
```

Comment:
Without this, I guess the only way to know which locking mode is on is via. the command line in the hs_err file.  It didn't really belong here, but maybe we want to find somewhere else for it.  Not with this change.  It's good to clean this up for this change.

Reply:
We print flags at the end which are non-default. But we do not know if someone changed them in a build. 
Should probably fix this somehow. 

Also printing all flag values, not just non-default ones would help.